.github/workflows/quay.yml

```
@@ -0,0 +1,23 @@
+name: quay
+
+on: push
+
+# on:
+#   push:
+#     branches:
+#       - master
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    env:
+      header: "Content-Type:application/json"
+      payload: "{\"commit\":\"${{ github.sha }}\",\"ref\":\"${{ github.ref }}\",\"default_branch\":\"master\"}"
+      url: "${{ secrets.QUAY_WEBHOOK_URL }}"
+    steps:
+    - name: debug
+      run: |
+        echo $header
+        echo $payload
+    - name: update container on quay.io
+      run: curl -H $header -d $payload $url
```

You may want to add some built-in bash set options so we explicitly fail if there are things like pipe failures or undefined variables.

```suggestion
    - name: update container on quay.io
      run: |
        set -eu
        curl -H $header -d $payload $url
```

---

You may need to add the `#! /bin/bash` shebang at the top too.

---

Bet will do